probe-rs/src/architecture/arm/core/mod.rs

```diff
@@ -286,7 +294,234 @@ static ARM_REGISTER_FILE: RegisterFile = RegisterFile {
     psp: Some(&register::PSP),
     extra: Some(&register::EXTRA),
     psr: Some(&register::XPSR),
-    // TODO: Floating point support
+
+    fpscr: Some(&register::FPSCR),
```

Comment:
This is an ARM specific name, could we make it more generic?  Maybe `fp_status`?